packages/core/src/connectivity/scp-cf/xsuaa-service.ts

```diff
@@ -292,5 +330,6 @@ function getInstanceCircuitBreaker(breaker?: any | undefined): any {
 const grantTypeMapper = {
   user_token: 'User token',
   refresh_token: 'Refresh token',
-  client_credentials: 'Client credentials'
+  client_credentials: 'Client credentials',
+  'urn:ietf:params:oauth:grant-type:jwt-bearer': 'JWT token'
```

Comment:
can you use the value of the GrantType enum here as key? @marikaner is this possible, then you have the string only in one place?